tools/data_builder.py

```diff
@@ -33,6 +33,10 @@
     r"(http|ftp|https):\/\/([\w\-_]+(?:(?:\.[\w\-_]+)+))" r"([\w\-\.,@?^=%&amp;:/~\+#\!]*[\w\-\@?^=%&amp;/~\+#\!])?"
 )
 
+GEOLOCATION_REGEX = re.compile(
+    r"^\(?([-+]?(?:[1-8]?\d(?:\.\d{6,7})?|90(?:\.0+)?)),\s*([-+]?(?:180(?:\.0+)?|(?:(?:1[0-7]\d)|(?:[1-9]?\d))(?:\.\d{6,7})?))\)?$"
```

Comment:
Regex could [probably be simplified](https://stackoverflow.com/questions/3518504/regular-expression-for-matching-latitude-longitude-coordinates)

Reply:
Using one of the regex values in there with a few requested pieces in addition to improve the accuracy

Reply:
Is it possible to at least break into 2 regex's for lat & long, since the result is being pulled apart as `result_1` and `result_2` anyway, before re-conjoining in return? This is just a lil' hard to look at/read 😵

I was gunna do a code suggestion but on my local machine, I can't seem to get baseline tests at 100% passage following the tools/README instructions. 7 tests fail, errors indicate something going wrong when trying to self-initialize this sqllite3 database. 🤷

Reply:
wut

I have never done that.

I don't even known how sqlite got in there

Reply:
Lmao I'm guessing it's a dependency that uses it then? Here's what I'm seeing

![image](https://user-images.githubusercontent.com/808746/94497877-8c10dd80-01c6-11eb-9cdf-e6ffe2a60074.png)